test/Microsoft.ML.Tokenizers.Tests/EnglishRobertaTests.cs

```diff
@@ -90,9 +90,15 @@ public async void TokenizationTest()
 
             try
             {
-                await Utils.DownloadFile(_vocabUrl, vocabFile);
-                await Utils.DownloadFile(_mergeUrl, mergeFile);
-                await Utils.DownloadFile(_dictUrl, translationFile);
+                bool result = await Utils.DownloadFile(_vocabUrl, vocabFile);
+                result = result && await Utils.DownloadFile(_mergeUrl, mergeFile);
+                result = result && await Utils.DownloadFile(_dictUrl, translationFile);
+
+                if (!result)
+                {
+                    // tolerate network issues
+                    return;
```

Comment:
This still makes me a bit nervous since we'll just silently lose coverage if those files go away.  I'm more inclined to keep the failure path (with retries to address flakiness).  Then if we find that the openai servers aren't reliable we mirror the data to our blob storage.